src/js/lineChartTimeSeriesArea.js

```diff
@@ -63,6 +63,9 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             removeArea: {
                 funcName: "floe.chartAuthoring.lineChart.timeSeries.area.removeArea",
                 args: ["{that}"]
+            },
+            transitionArea: {
+                funcName: "floe.chartAuthoring.lineChart.timeSeries.area.updateArea.paginateTransition"
```

Comment:
The invoker is hooked up with `paginateTransition` function by default while the other transition function is named `defaultTransition`. Can we have a better name for the latter, and add a code comment to describe these two alternative transition functions and what they are for. 

The same comment applies to the usage of `paginateTransition` and `defaultTransition` in other js files: `lineChartTimeSeriesLine.js` and `lineChartTimeSeriesPoints.js`.